process.rb

```diff
@@ -28,6 +28,28 @@ def slugify(word)
   (word || '').downcase.gsub(/[\._~!$&'()+,;=@]+/, '').gsub(/[^a-z0-9-]+/, '-')
 end
 
+# Sort like:
+# 1. Mayor
+# 2. City Council ...
+# 3. City ...
+# 4. School Board
+#
+# If multiple offices match the same regex they are sorted alphabetically (i.e.
+# "School Board District 1", then "School Board District 2")
+SORT_PATTERNS = [
+  /mayor/i,
+  /city council/i,
+  /city /i,
```

Comment:
I agree city-wide offices should come first. That's the way it is in Elina's mock-ups.

Reply:
Counter point: I feel pretty strongly that the City Attorney race is less interesting to users than their council district race.

That said, I agree that it's more intuitive to group all the city-wide elections together. I'll make the switch but I think this might be worth keeping in mind.